### ISSUE TITLE
Contributor import fails

### ISSUE THREAD

**Issue by janno42** [fsr-itse/EvaP#951]
The contributor import fails with an Internal Server Error when the Excel file contains a user who was already added as a contributor before.

```
Internal Server Error: /staff/semester/.../course/.../person_import

IntegrityError at /staff/semester/.../course/.../person_import
duplicate key value violates unique constraint "evaluation_contribution_course_id_..._uniq"
DETAIL:  Key (course_id, contributor_id)=(..., ...) already exists.
```

In this situation the importer should show a warning, skip the user from the import file and keep the existing contributor.

**Comment by He3lixxx** [comment]
This should get fixed when implementing #724 / importing #948, since the list of users that should be added will get filtered there. Nevertheless, there should be tests covering this.

**Comment by karyon** [comment]
i'm preeeeetty sure this is fixed.
